src/styles/blocks/month.scss

```diff
@@ -0,0 +1,57 @@
+.month {
+  display: flex;
+  flex-basis: $calendar-width;
+  min-width: $calendar-width;
+  flex-wrap: wrap;
+  align-content: center;
+  padding: $padding-size;
+  gap: $gap-size;
+
+  @for $i from 1 through $max-days-in-month {
+    &--day-#{$i} {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      width: $cell-size;
+      height: $cell-size;
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: $number-size;
+      background-color: #eee;
+      border: 1px solid black;
```

Comment:
These styles and hover styles should not be in the loop. They refer to all day-boxes.

Use loop to only add numbers to the boxes.